opencontrail/map.jinja

```diff
@@ -79,7 +84,7 @@ database:
 web:
   Debian:
     pkgs:
-         ['contrail-web-controller', 'nodejs-legacy', 'gettext-base']
```

Comment:
After investigation, I have removed the 'nodejs-legacy' package completely, because 'nodejs' is already installed as dependency of 'contrail-web-controller'.